### PR TITLE
Pillbottles can now be made

### PR DESCRIPTION
**Pull request by ghost**
:cl: Mark9013100
add: Pill bottles can now be produced in the autolathe.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

As far as I can tell, there's no method of getting more pill bottles ingame, so I added a recipe for them in the autolathe, takes 20 metal, 100 glass.

### CODE REVIEW

**Comment by 81Denton** [comment]
Very nice

**Comment by arsserpentarium** [comment]
Make it so for space cleaner spray too.

**Comment by XDTM** [comment]
Make it a lathe recipe

**Comment by arsserpentarium** [comment]
No. Crafting is funnier.

**Comment by XDTM** [comment]
It is not, and the sheet to bottle ratio doesn't make sense, considering two sheets of glass make 8 large beakers.

**Comment by Ratvar** [comment]
The 5 sheets to rolling chair ratio doesn't make sense, as walls are made from 4 sheets. Soo?

**Comment by Mark9013100** [comment]
@arsserpentarium Seems a bit too OP to let chemistry get spray bottles without leaving the lab, sides they can be ordered from cargo.

@XDTM Eh, not really sure it'd make sense for pill bottles to be available in the autolathe.

**Comment by XDTM** [comment]
Why not? Beakers can be autolathed, and they have more uses than pill bottles. And unlike sprays there's no real balance concern, i'd be surprised if anyone actually used them.

**Comment by Mark9013100** [comment]
Fair point about balance, but I can't really see pill bottles being made of glass/metal. Sides, all the more reason to make plastic.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -500,6 +500,7 @@ GLOBAL_LIST_INIT(plastic_recipes, list(
 	new /datum/stack_recipe("plastic flaps", /obj/structure/plasticflaps, 5, one_per_turf = TRUE, on_floor = TRUE, time = 40), \
 	new /datum/stack_recipe("water bottle", /obj/item/reagent_containers/glass/beaker/waterbottle/empty), \
 	new /datum/stack_recipe("large water bottle", /obj/item/reagent_containers/glass/beaker/waterbottle/large/empty,3), \
+	new /datum/stack_recipe("pill bottle", /obj/item/storage/pill_bottle,2), \
```
Comment (Jalleo) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: personally I think this would be better in a autolathe than a crafting recipe on a stack.

**Comment by Spaceinaba** [comment]
why cant we just make autolathes accept plastic

**Comment by praisenarsie** [comment]
only metal and glass, autolathe needs a rework to be less fucking stupid

**Comment by Bawhoppen** [comment]
Autolathe does need a rework

**Comment by Bawhoppen** [comment]
For now just make them use 1 plastic sheet since that makes a little more sensr

**Comment by Jalleo** [comment]
It is easy to make the autolathe accept more materials I will do it tomorrow or later tonight

**Comment by PKPenguin321** [comment]
Honestly would it really be that big of a deal to allow chemmasters to produce them?

**Comment by Bawhoppen** [comment]
@jalleo Needs more of a rework, stuff like it needing chems for certain recipes is also necessary

@pkpenguin321 Only if we add a plastic material storage to have them be made with (same with their glass bottles aswell)

I am personally not fond of stuff being synthesized out of nowhere, if you couldn't tell

**Comment by JJRcop** [comment]
What's holding this PR up?

**Comment by duncathan** [comment]
it's stupid that these are crafted from a stack and even stupider that it takes two sheets for something so benign imo

**Comment by 81Denton** [comment]
@JJRcop Nitpicking and a lack of clear design guidelines
Otherwise something as trivial as making tiny pill bottles wouldn't be held up for a week

**Comment by XDTM** [comment]
What dunc said, and there are plastic items in autolathes already anyway

**Comment by Mark9013100** [comment]
Fine, I'll throw it in the autolathe.

**Comment by ExcessiveUseOfCobblestone** [comment]
When do people use these outside of chemistry-related?

**Pull request merged**